### PR TITLE
Add command to download precomputed stuff

### PR DESCRIPTION
**Pull request by smondet**
E.g. from the docker container:

```
    bash ./disco.sh full-up-cache gs://some-storage/b37_precomputed.tgz reference-genome
```

### CODE REVIEW

**Comment by smondet** [comment]
Where a targz `b37decoy.tgz` would look like:

```
b37decoy/
b37decoy/b37decoy.fasta.ann
b37decoy/b37decoy.fasta.gz
b37decoy/transcripts.gtf
b37decoy/transcripts.gtf.gz
b37decoy/b37decoy.fasta.sa
b37decoy/b37decoy.fasta.bwt
b37decoy/star-index/
b37decoy/star-index/Genome
b37decoy/star-index/chrNameLength.txt
b37decoy/star-index/chrLength.txt
b37decoy/star-index/SAindex
b37decoy/star-index/transcriptInfo.tab
b37decoy/star-index/sjdbInfo.txt
b37decoy/star-index/exonInfo.tab
b37decoy/star-index/exonGeTrInfo.tab
b37decoy/star-index/chrStart.txt
b37decoy/star-index/genomeParameters.txt
b37decoy/star-index/sjdbList.out.tab
b37decoy/star-index/SA
b37decoy/star-index/geneInfo.tab
b37decoy/star-index/sjdbList.fromGTF.out.tab
b37decoy/star-index/chrName.txt
b37decoy/b37decoy.fasta.amb
b37decoy/b37decoy.dict
b37decoy/cosmic.vcf
b37decoy/dbsnp.vcf.idx
b37decoy/b37decoy.fasta.fai
b37decoy/dbsnp.vcf.gz
b37decoy/dbsnp.vcf
b37decoy/b37decoy.fasta
b37decoy/b37decoy.fasta.pac
```

File: docs/disco.sh [hunk 1]
```diff
@@ -176,6 +177,30 @@ create-nfs () {
...
+    gsutil version -l
+    
+    sudo apt-get install -y gcc python-dev python-setuptools
+    sudo easy_install -U pip
+    #pip uninstall crcmod
+    sudo pip install -U crcmod
+    gsutil -m cp $url \
+        $BIOKEPI_WORK_DIR/$dir/$(basename $url)
+    ( cd $BIOKEPI_WORK_DIR/$dir/ ; tar xvfz $(basename $url) )
+    chmod -R 777 /nfs-pool/
```
Comment (ihodes) [inline docs/disco.sh:202]: Might need sudo?

File: docs/disco.sh [hunk 2]
```diff
@@ -176,6 +177,30 @@ create-nfs () {
...
+    fi
+
+    echo "Getting a precomputed $dir from $url"
+    mkdir -m a+rwx -p $BIOKEPI_WORK_DIR/4dir
+
+    gsutil version -l
+    
+    sudo apt-get install -y gcc python-dev python-setuptools
+    sudo easy_install -U pip
+    #pip uninstall crcmod
```
Comment (ihodes) [inline docs/disco.sh]: can probably just delete this line?

**Comment by ihodes** [comment]
maybe worth -z "$BIOKEPI_WORK_DIR" to warn that config must be sourced, since you're not doing that automatically?

**Pull request merged**